src/Schema/Directives/Fields/FieldMiddleware.php

```diff
@@ -1,18 +1,12 @@
 <?php
 
-namespace Nuwave\Lighthouse\Support\Contracts;
+namespace Nuwave\Lighthouse\Schema\Directives\Fields;
 
```

Comment:
Let's not mix the contracts with the classes, so let's move these back to the `Contracts` namespace (which should also help w/ breaking changes)

Reply:
The reason why i did it was to make it clear which kind of directive each of the Interfaces belongs to. I am all for avoiding unnecessary breaking changes though, is there a way to get the best of both?

Reply:
I think `FieldResolver` and `FieldMiddleware` clarify that they belong to a field and `ArgMiddlware` to and arg and so on. I wouldn't mind restructuring some of the namespaces but I'd prefer to do that a bit down the line as some users are getting used to the overhaul of `v2.0` and I'd hate to have them encounter any more breaking changes than they need to.

Reply:
I will make it so!